examples/region_sharding/lib/utils.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
```

Comment:
so should we be able to remove this file now that we have `examples/common/lib/utils.sh`?

Reply:
Doh! Yeah, those should already be gone. Thanks!